NEWS

```diff
@@ -1,6 +1,11 @@
 shiny-server 1.5.0
 --------------------------------------------------------------------------------
-* Add auto-reconnect capabilities.
+* Add auto-reconnect capabilities. Can be disabled via `disable_reconnect`
```

Comment:
Maybe then just being explicit is sufficient. "Reconnect can be enabled by setting `disable_reconnect` to a value `false`."